raiden/raiden_service.py

```diff
@@ -373,7 +373,9 @@ def _callback_new_block(self, current_block_number, chain_id):
         # expected side-effects are properly applied (introduced by the commit
         # 3686b3275ff7c0b669a6d5e2b34109c3bdf1921d)
         with self.event_poll_lock:
-            for event in self.blockchain_events.poll_blockchain_events():
+            for event in self.blockchain_events.poll_blockchain_events(
+                self.get_block_number(),
```

Comment:
This must use `current_block_number`, otherwise the logs of the current event wont be polled, and on restart we will lose events of the latest block.